@teoli2003, @yin1999, @schalkneethling Since we moved to using GitHub APIs to get the changed files in a PR we've started to exceed the API call limit:
> gh: API rate limit exceeded for installation ID 5537481. (HTTP 403)
> Error: Process completed with exit code 1.
> https://github.com/mdn/content/actions/runs/5029421367/jobs/9021063984?pr=26862

As per the GitHub docs 
> Rate limits also depend on whether the GitHub App authenticates with a user access token or with an installation access token.

--- a/files/en-us/web/api/webtransport/index.md
+++ b/files/en-us/web/api/webtransport/index.md
@@ -45,7 +45,7 @@ The **`WebTransport`** interface of the {{domxref("WebTransport API", "WebTransp
   - : Asynchronously opens a bidirectional stream ({{domxref("WebTransportBidirectionalStream")}}) that can be used to read from and write to the server.
 - {{domxref("WebTransport.createUnidirectionalStream", "createUnidirectionalStream()")}}
   - : Asynchronously opens a unidirectional stream ({{domxref("WritableStream")}}) that can be used to write to the server.
-- {{domxref("WebTransport.getStats", "getStats()")}}
+- {{domxref("WebTransport.getStats", "getStats()")}} {{Experimental_Inline}}
   - : Asynchronously returns a {{jsxref("Promise")}} that fulfills with an object containing HTTP/3 connection statistics.
 
 ## Examples

--- a/files/en-us/web/api/webtransportreceivestream/index.md
+++ b/files/en-us/web/api/webtransportreceivestream/index.md
@@ -30,7 +30,7 @@ _Inherits properties from its parent interface, {{domxref("ReadableStream")}}._
 
 _Also inherits properties from its parent interface, {{domxref("ReadableStream")}}._
 
-- {{domxref("WebTransportReceiveStream.getStats()")}}
+- {{domxref("WebTransportReceiveStream.getStats()")}} {{Experimental_Inline}}
   - : Returns a {{jsxref("Promise")}} that resolves with statistics related to this stream.
 
 ## Examples

--- a/files/en-us/web/html/element/link/index.md
+++ b/files/en-us/web/html/element/link/index.md
@@ -259,7 +259,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
     The value of the attribute should be a MIME type such as **text/html**, **text/css**, and so on.
     The common use of this attribute is to define the type of stylesheet being referenced (such as **text/css**), but given that CSS is the only stylesheet language used on the web, not only is it possible to omit the `type` attribute, but is actually now recommended practice.
     It is also used on `rel="preload"` link types, to make sure the browser only downloads file types that it supports.
-- `blocking`
+- `blocking` {{Experimental_Inline}}
   - : This attribute explicitly indicates that certain operations should be blocked on the fetching of an external resource. The operations that are to be blocked must be a space-separated list of blocking attributes listed below.
     - `render`: The rendering of content on the screen is blocked.
 

--- a/files/en-us/web/html/element/script/index.md
+++ b/files/en-us/web/html/element/script/index.md
@@ -142,7 +142,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
         Developers must use a valid MIME type that is not a JavaScript MIME type to denote data blocks.
         All of the other attributes will be ignored, including the `src` attribute.
 
-- `blocking`
+- `blocking` {{Experimental_Inline}}
   - : This attribute explicitly indicates that certain operations should be blocked on the fetching of the script. The operations that are to be blocked must be a space-separated list of blocking attributes listed below.
     - `render`: The rendering of content on the screen is blocked.
 

--- a/files/en-us/web/html/element/style/index.md
+++ b/files/en-us/web/html/element/style/index.md
@@ -27,7 +27,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
   - : A cryptographic nonce (number used once) used to allow inline styles in a [style-src Content-Security-Policy](/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src). The server must generate a unique nonce value each time it transmits a policy. It is critical to provide a nonce that cannot be guessed as bypassing a resource's policy is otherwise trivial.
 - `title`
   - : This attribute specifies [alternative style sheet](/en-US/docs/Web/CSS/Alternative_style_sheets) sets.
-- `blocking`
+- `blocking` {{Experimental_Inline}}
   - : This attribute explicitly indicates that certain operations should be blocked on the fetching of critical subresources. [`@import`](/en-US/docs/Web/CSS/@import)-ed stylesheets are generally considered as critical subresources, whereas [`background-image`](/en-US/docs/Web/CSS/background-image) and fonts are not.
     - `render`: The rendering of content on the screen is blocked.
 

